core/pom.xml

```diff
@@ -58,7 +58,6 @@ limitations under the License.
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
-      <scope>test</scope>
```

Comment:
Why this change? It doesn't look like you're changing anything relating to commons-lang3 in the code.